### DISCUSSION TITLE
JRPC url error 502

### DISCUSSION THREAD

**Discussion by locpx139** [Web3Auth/Web3Auth#1171] [Help]
This node alway get error 502

INFO  Response: 502
 INFO  Url: https://torus-node.ens.domains/jrpc
 INFO  Response: 502
 INFO  Url: https://torus-node.ens.domains/jrpc
 INFO  Response: 502
 INFO  Url: https://torus-node.ens.domains/jrpc

**Comment by shang-chi-7** [comment]
Any solution found for this ? @locpx139
